content/root/how-ably-works.textile

```diff
@@ -14,7 +16,7 @@ jump_to:
     - Any internet device
 ---
 
-Ably's global realtime service enables Internet enabled devices, such as a browser, phone, server or IoT sensor, to stream data in realtime between to any other Internet connected device in milliseconds. The Ably platform brings enterprise scale messaging to developers by delivering a "highly-available service":https://knowledge.ably.com/ablys-uptime-guarantee, "message delivery guarantees":https://knowledge.ably.com/message-durability-and-qos-quality-of-service and "low-latencies globally":https://knowledge.ably.com/round-trip-latency-and-performance (typically less than 60ms).
+Ably's realtime platform enables data to be streamed between Internet connected devices such as browsers, phones, servers and IoT (Internet of Things) devices in milliseconds. The Ably platform provides developers with enterprise scale messaging by delivering a "highly-available service":https://knowledge.ably.com/ablys-uptime-guarantee, "message delivery guarantees":https://knowledge.ably.com/message-durability-and-qos-quality-of-service and "globally low-latencies":https://knowledge.ably.com/round-trip-latency-and-performance that are typically less than 60ms.
```

Comment:
> Ably's realtime platform enables data to be streamed between Internet connected devices such as...

Internet-connected?

> "globally low-latencies":https://knowledge.ably.com/round-trip-latency-and-performance that are typically less than 60ms.

... and low latency on a global basis. "Round trip latencies":https://knowledge.ably.com/round-trip-latency-and-performance are typically less than 60ms.

Reply:
Added hyphen and reworded the sentence, thanks!